vars/copyDockerImage.groovy

```diff
@@ -0,0 +1,21 @@
+void call(Map args = [:]){
+
+    sh "docker logout"
+
+    if(args.destinationType == 'docker'){
+        withCredentials([usernamePassword(credentialsId: args.destinationCredentialIdentifier, usernameVariable: 'DOCKER_USERNAME', passwordVariable: 'DOCKER_PASSWORD')]) {
+            sh """
+                docker login -u $DOCKER_USERNAME -p $DOCKER_PASSWORD
+                gcrane cp ${args.sourceImagePath} ${args.destinationImagePath}
+            """
+        }
+    } else if(args.destinationType == 'ecr'){
+        withAWS(role: 'Upload_ECR_Image', roleAccount: "${args.accountName}", duration: 900, roleSessionName: 'jenkins-session') {
+            sh """
+                aws ecr-public get-login-password --region us-east-1 | docker login --username AWS --password-stdin ${args.destinationCredentialIdentifier}
```

Comment:
Is it always going to be `us-east-1` or should we take this as a param? Your call

Reply:
I have the repositories created on `us-east-1` as of today